prboom2/src/r_draw.c

```diff
@@ -587,3 +590,16 @@ int R_GetFuzzPos()
 {
   return fuzzpos;
 }
+
+void R_ResetFuzzCol(int height)
+{
+  R_ResetColumnBuffer();
+
+  fuzzpos = (fuzzpos + (height / fuzzcellsize)) % FUZZTABLE;
+}
+
+void R_CheckFuzzCol(int x, int height)
+{
+  if (!(x % fuzzcellsize))
+    R_ResetFuzzCol(height);
+}
```

Comment:
No "newline" at the end of the file. A small thing, but it can be annoying https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline